docs/docs/examples/cookbooks/toolhouse_llamaindex.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
The required tools for this example are installed by default in the user's Toolhouse dashboard. Regardless, the notebook already constains instructions on how to install such tools. This is useful in case the user wants to try to uninstall and reinstall a tool.